src/esp/sim/Simulator.h

```diff
@@ -719,6 +731,25 @@ class Simulator {
    */
   bool isNavMeshVisualizationActive();
 
+  /**
+   * @brief Return a ref to a new drawables in the currently active scene, for
+   * object creation.  Eventually support multi-scene ID
+   * @param sceneID The scene to get the drawables for.  Currently not used.
+   */
+  inline esp::gfx::DrawableGroup& getDrawables(
```

Comment:
Minor naming suggestion.
```suggestion
  inline esp::gfx::DrawableGroup& getDrawableGroup(
```